systemd-sslh-generator.c

```diff
@@ -55,9 +55,13 @@ static int get_listen_from_conf(const char *filename, char **listen[]) {
                             config_setting_source_line(addr));
                     return -1;
                 } else {
+                    char *resolved_listen;
+
                     (*listen)[i] = malloc(strlen(resolve_listen(hostname, port)));
```

Comment:
doesn't `resolve_listen()` leak here as well?

Reply:
(basically it seems to me you want to assign `resolved_listen`  once, then free it.)

Reply:
true, overlooked that one